lib/trino-hive-formats/src/test/java/io/trino/hive/formats/ion/TestIonFormat.java

```diff
@@ -210,25 +221,45 @@ public void testOversizeOrOverpreciseDecimals()
     public void testEncode()
             throws IOException
     {
-        List<Column> columns = List.of(
-                new Column("magic_num", INTEGER, 0),
-                new Column("some_text", VARCHAR, 1),
-                new Column("is_summer", BooleanType.BOOLEAN, 2),
-                new Column("byte_clob", VarbinaryType.VARBINARY, 3),
-                new Column("sequencer", new ArrayType(INTEGER), 4),
-                new Column("struction", RowType.rowType(
-                        field("foo", INTEGER),
-                        field("bar", VARCHAR)), 5));
-
         List<Object> row1 = List.of(17, "something", true, new SqlVarbinary(new byte[] {(byte) 0xff}), List.of(1, 2, 3), List.of(51, "baz"));
         List<Object> row2 = List.of(31, "somebody", false, new SqlVarbinary(new byte[] {(byte) 0x01, (byte) 0xaa}), List.of(7, 8, 9), List.of(67, "qux"));
         String ionText = """
                 { magic_num:17, some_text:"something", is_summer:true, byte_clob:{{/w==}}, sequencer:[1,2,3], struction:{ foo:51, bar:"baz"}}
                 { magic_num:31, some_text:"somebody", is_summer:false, byte_clob:{{Aao=}}, sequencer:[7,8,9], struction:{ foo:67, bar:"qux"}}
                 """;
 
-        Page page = toPage(columns, row1, row2);
-        assertIonEquivalence(columns, page, ionText);
+        Page page = toPage(TEST_COLUMNS, row1, row2);
+        assertIonEquivalence(TEST_COLUMNS, page, ionText);
+    }
+
+    @Test
+    public void testEncodeWithNullField()
+            throws IOException
+    {
+        List<Object> row1 = Arrays.asList(null, "something", true, new SqlVarbinary(new byte[] {(byte) 0xff}), List.of(1, 2, 3), List.of(51, "baz"));
+        List<Object> row2 = Arrays.asList(31, "somebody", null, new SqlVarbinary(new byte[] {(byte) 0x01, (byte) 0xaa}), List.of(7, 8, 9), List.of(67, "qux"));
+        String ionText = """
```

Comment:
nit: it's not super obvious what is expected to be missing from this test. Were it me I would either use less columns, or just have all of them be null.

Reply:
Yeah I agree, I will update the tests with more clear test cases.